my_form.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Cam's Quiz</title>
+  <link rel="stylesheet" href="my_style.css">
+  <script src="nav.js"></script>
+  <script src="form_validate.js"></script>
+</head>
+
+<body onload="setNav(window.location.pathname)">
+  <nav id="main-nav"></nav>
+
+  <main>
+    <h1>What Type of Student Are You?</h1>
+
+    <form id="quizForm" onsubmit="return validate(event)">
+      <fieldset>
+        <legend>Find Out Your Student Type!</legend>
+
+     
+        <label for="name">Your Name:</label><br>
+        <input type="text" id="name" name="name"><br><br>
+
+  
+        <label for="email">Your Email:</label><br>
+        <input type="email" id="email" name="email"><br><br>
+
+   
+        <p>1. How do you study for exams?</p>
+        <label><input type="radio" name="study" value="early"> I start a week before</label><br>
+        <label><input type="radio" name="study" value="last_minute"> The night before</label><br>
+        <label><input type="radio" name="study" value="never"> I don't study</label><br><br>
+
+   
+        <p>2. Favorite study snack:</p>
+        <label><input type="checkbox" name="snack" value="chips"> Chips</label>
+        <label><input type="checkbox" name="snack" value="fruit"> Fruit</label>
+        <label><input type="checkbox" name="snack" value="candy"> Candy</label><br><br>
+
+    
+        <p>3. How many hours of sleep do you get?</p>
+        <input type="number" id="sleep" name="sleep" min="0" max="12"><br><br>
+
+     
+        <p>4. Choose your dream profession of these options:</p>
+        <label><input type="checkbox" name="profession" value="lawyer"> Lawyer</label>
+		<label><input type="checkbox" name="profession" value="professional_athlete"> Professional Athlete</label>
+	    <label><input type="checkbox" name="profession" value="doctor"> Doctor</label><br><br>
+
+
+     
+        <p>5. Do you plan on doing a masters?</p>
+        <label><input type="radio" name="masters" value="yes"> Yes</label>
+        <label><input type="radio" name="masters" value="no"> No</label><br><br>
+
+        <button type="submit">Submit Quiz</button>
+      </fieldset>
+    </form>
+  </main>
```

Comment:
Your footer hides part of the submit button. An easy fix would be to add a margin-bottom to your "main" here.